js/exporter/jspdf/v3/rows_splitting.js

```diff
@@ -1,24 +1,25 @@
 import { isDefined } from '../../../core/utils/type';
+import { round } from './draw_utils';
 
-function splitRectsByPages(rects, topLeft, maxBottomRight) {
+function splitRectsByPages(rects, margin, topLeft, maxBottomRight) {
     if(!isDefined(rects) || rects.length === 0) { // Empty Table
         return [[]];
     }
 
-    const rectsByPage = splitRectsHorizontalByPages(rects, topLeft, maxBottomRight);
+    const rectsByPage = splitRectsHorizontalByPages(rects, margin, topLeft, maxBottomRight);
     // TODO: splitRectsVerticalByPages
 
     return rectsByPage;
 }
 
-function splitRectsHorizontalByPages(rects, topLeft, maxBottomRight) {
+function splitRectsHorizontalByPages(rects, margin, topLeft, maxBottomRight) {
     const pages = [];
     const rectsToSplit = [...rects];
 
     while(rectsToSplit.length > 0) {
         let currentPageMaxRectRight = 0;
         const currentPageRects = rectsToSplit.filter(rect => {
-            const currentRectRight = Math.round((rect.x + rect.w) * 1000) / 1000;
+            const currentRectRight = round(rect.x + rect.w);
```

Comment:
It's not clear why we don't need math.round now, it was before:

const currentRectRight = Math.round((rect.x + rect.w) * 1000) / 1000;

Reply:
We have identical code in 'draw_utils.js' file:
https://github.com/DevExpress/DevExtreme/blob/22_1/js/exporter/jspdf/v3/draw_utils.js#L8
In this place, I got rid of duplicate code.